alacritty_terminal/src/term/mod.rs

```diff
@@ -425,9 +425,11 @@ impl<'a, C> Iterator for RenderableCellsIter<'a, C> {
 
                     // Apply cursor color, or invert the cursor if it has a fixed background close
                     // to the cell's background.
-                    match self.cursor.cursor_color {
-                        CellRgb::Rgb(color) if color.contrast(cell.bg) < MIN_CURSOR_CONTRAST => (),
-                        _ => cell.fg = self.cursor.cursor_color.color(cell.fg, cell.bg),
+                    if !matches!(
+                        self.cursor.cursor_color,
+                        CellRgb::Rgb(color) if color.contrast(cell.bg) < MIN_CURSOR_CONTRAST)
+                    {
+                        cell.fg = self.cursor.cursor_color.color(cell.fg, cell.bg);
                     }
```

Comment:
This thing was formatted manually, since formatter doesn't like macros in general, and so ignore those lines.

Reply:
> so ignore those lines.

That's no reason to ignore them. The formatting is still off. Probably a good idea to turn that macro into a function and try to format that.